apps/hassentity/hass_entity.star

```diff
@@ -79,15 +79,15 @@ def get_schema():
         version = "1",
         fields = [
             schema.Text(
-                id = "ha_ip",
-                name = "Home Assistant External Url",
-                desc = "External HA Url (and optional port). Ex. https://abc.ui.nabu.casa or https://hass.mydomain:8123",
+                id = "nabu_casa_url_key",
+                name = "Nabu Casa Url Key",
+                desc = "The random letters and numbers in your Nabu Casa URL. Ex. Input 'abc123' for this nabu casa url https://abc123.ui.nabu.casa",
```

Comment:
Just a heads up that the `desc` fields is currently not visible to users in the Tidbyt mobile app.